blogs_notebooks/titanic_data_science_solutions_blog_1.ipynb

```diff
@@ -0,0 +1,492 @@
+{
```

Comment:
<p>let's make this more conversational "cooking show" style instead of step by step/captioning.  If we want to do step by step, we'll need step numbers.  Remember, they are following you and sometimes needs to know "Why"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai-community/showcase/pull/56/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>